Cargo.toml

```diff
@@ -68,13 +60,13 @@ default = [
 ]
 runtime = [
     "tcp",
-    "tokio",
+    "tokio/time",
 ]
 tcp = [
     "net2",
-    "tokio-executor/blocking",
-    "tokio-net",
-    "tokio-timer",
+    "tokio/blocking",
+    "tokio/tcp",
+    "tokio/time",
 ]
 
 # unstable features
```

Comment:
Should `unstable-stream` be renamed to `stream` for the same reasons as in H2? https://github.com/hyperium/h2/pull/433

Reply:
Yes, though I'd rather do it in a separate PR.